docs/source/tutorial.rst

```diff
@@ -280,37 +280,25 @@ To use a registered domain in your relayer, type its name in relayer config.
 
 From command line
 ^^^^^^^^^^^^^^^^^
-First, prepare ``.env`` file in universal-login-ops directory.
-
-Parameters:
-  - **JSON_RPC_URL** : string - JSON-RPC URL of an Ethereum node
-  - **PRIVATE_KEY** : string - private key to execute registrations. `Note:` You need to have ether on it to pay for contracts deployment.
-  - **ENS_ADDRESS** : string - the address of an ENS contract
-  - **PUBLIC_RESOLVER_ADDRESS** : string - the address of a public resolver. For the Ropsten test network a working public resolver address is ``0x4C641FB9BAd9b60EF180c31F56051cE826d21A9A`` and for the Rinkeby test network a public resolver address is ``0x5d20cf83cb385e06d2f2a892f9322cd4933eacdc``.
-
-  Example ``.env`` file:
-
-  ::
-
-    JSON_RPC_URL='https://ropsten.infura.io'
-    PRIVATE_KEY='YOUR_PRIVATE_KEY'
-    ENS_ADDRESS='0x112234455c3a32fd11230c42e7bccd4a84e02010'
-    PUBLIC_RESOLVER_ADDRESS='0x4C641FB9BAd9b60EF180c31F56051cE826d21A9A'
 
 To register an test ENS domain type in the console:
 
   ::
 
-    universal-login register:test:domain my-domain
+    universal-login register:test:domain [my-domain] [ensAddress] [publicResolverAddress] --privateKey [privateKey] --nodeUrl [url]
 
 Parameters:
   - **my-domain** - a domain to register
+  - **ensAddress** : string - the address of an ENS contract
```

Comment:
([list of ENS addresses](https://docs.ens.domains/ens-deployments) on public networks)